storage-inventory/ops/README.md

```diff
@@ -128,139 +179,318 @@ The [`CADC Direct Data Service`](https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/en
     - See the [opencadc registry server](https://github.com/opencadc/reg/tree/master/cadc-registry-server) documentation for configuarion details.
-    - `resourceIDs`
+    - [`resourceIDs`](#term-resourceid)
         - you will need to choose resourceIDs for services and resources that you deploy, and which need to be referenced by other services and applications.  For example, if your `minoc` service is available at the URL `https://www.example.org/minoc` and you choose a resourceID of `ivo://example.org/minoc`, the registry config for that resource (in the `reg-resource-caps.properties` file for the registry service) would look like:
-            > ivo://example.org/minoc = `https://www.example.org/minoc`
-
-            This resourceID will appear in, for example, the `minoc.properties` file in the `minoc` service config:
-            > org.opencadc.minoc.resourceID = ivo://example.org/minoc
+        ```
+                ivo://example.org/minoc = https://www.example.org/minoc
+        ```
+        This resourceID will appear in, for example, the `minoc.properties` file in the `minoc` service config:
+        ```
+                org.opencadc.minoc.resourceID = ivo://example.org/minoc
+        ```
     - test with, e.g., `curl https://www.example.org/reg/resource-caps`
 
-1. **baldur**
-
-    - Container image: Use the latest `storage-inventory/baldur` image from `images.opencadc.org`
-    - See the [opencadc storage inventory baldur](https://github.com/opencadc/storage-inventory/tree/master/baldur) documentation for configuarion details.
+1. <a id='configuration-baldur'>**baldur**</a> - Permission service
+
+    - Required for: Storage site and Global site (Note: would likely be the same baldur instance for both)
+    - Container image: Use the latest `storage-inventory/baldur` image from [`images.opencadc.org`](#resource-repository)
+    - Uses an IVOA compatible GMS service and configured [namespaces](#term-namespace) to determine file access permissions.
+    - Configuration notes:
+        - `baldur.properties`:
+            - The `org.opencadc.baldur.allowedUser` x509 DN specified here is generally a 'service' user -- the services that call `baldur` need to be configured with this user's certificate.
+                - in `minoc` and `raven` coniguration, this is the `cadcproxy.pem` file for these services.
+            - The `org.opencadc.baldur.allowedGroup` is an IVOA GMS group [resourceID](#term-resourceid).  
+                - The GMS service must be registered in the available Registry service.
+                - the configured `readOnlyGroup` and `readWriteGroup` entries are also IVOA GMS group [resourceIDs](#term-resourceid).
+    - See the [opencadc storage inventory baldur](https://github.com/opencadc/storage-inventory/tree/master/baldur) documentation for more configuration details.
```

Comment:
Shouldn't this be the first bullet, i.e. the up-to-date, complete configuration is at this location and following is just some practical consideration. This is about maintenance and stating up front where the ultimate source resides.

Reply:
I was kinda thinking that, too.

Reply:
Moved these urls near to the top of the service sections